game/src/main/java/game/player/PlayerCommandProcessor.java

```diff
@@ -0,0 +1,42 @@
+package game.player;
+
+import engine.command.Command;
+import game.exception.CorruptPlayerCommandException;
+import engine.exception.WrongObjectIdException;
+import engine.geometry.Direction;
+import game.command.TankMoveCommand;
+import game.object.Tank;
+
+public class PlayerCommandProcessor {
```

Comment:
According to public methods this is not Processor class. This is more likely to be Factory